internal/cmd/config/list/list.go

```diff
@@ -35,56 +42,85 @@ func NewCmd(p *print.Printer) *cobra.Command {
 			examples.NewExample(
 				`List your active configuration`,
 				"$ stackit config list"),
+			examples.NewExample(
+				`List your active configuration in a json format`,
+				"$ stackit config list --output-format json"),
 		),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			configData := viper.AllSettings()
 
-			// Sort the config options by key
-			configKeys := make([]string, 0, len(configData))
-			for k := range configData {
-				configKeys = append(configKeys, k)
+			model, err := parseInput(p, cmd)
+			if err != nil {
+				return err
 			}
-			sort.Strings(configKeys)
+			return outputResult(p, model.OutputFormat, configData)
+		},
+	}
+	return cmd
+}
 
-			table := tables.NewTable()
-			table.SetHeader("NAME", "VALUE")
-			for _, key := range configKeys {
-				value := configData[key]
+func parseInput(p *print.Printer, cmd *cobra.Command) (*inputModel, error) {
```

Comment:
this function doesnt need to return error

Reply:
Removed the error from return values